src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1726,10 +1727,10 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         then: "Bidder request skipped flag should be false"
         def bidderRequest = bidder.getBidderRequests(bidRequest.id).last()
         verifyAll {
-            !bidderRequest.ext?.prebid?.floors?.skipped
+            BooleanUtils.isFalse(bidderRequest.ext?.prebid?.floors?.skipped)
```

Comment:
AFAIK it should be `bidderRequest.ext?.prebid?.floors?.skipped == false`

Reply:
Yes, it can be `bidderRequest.ext?.prebid?.floors?.skipped == false` but when we use this approach IntelliJ highlights this so I chose `BooleanUtils`.

Reply:
IDEA's warning is bogus in this case. For visibility's sake, we are better off explicit. Boolean utils will still coerce to false if the value is null. And we want to make sure that there is actually a false there.

And you also imported boolean utils from a wrong package too :)